libs/core/src/ports/index.ts

```diff
@@ -81,3 +82,19 @@ export const cache = port(function cache(cache?: Cache) {
     }
   );
 });
+
+/**
+ * Analytics port factory
+ */
+
+export const analytics = port(function analytics(analytics?: Analytics) {
+  return (
+    analytics || {
+      name: 'in-memory-analytics',
+      dispose: () => Promise.resolve(),
+      track: (event: string, payload: AnalyticsOptions) => {
```

Comment:
Shouldn't we be running this through the logging adapter instead of hardcoding in console? Or maybe it violates some hexagonal architecture pattern to use different adapters in unrelated ports?

Reply:
we can use the logger port here, but don't see the need to "log" stuff when using the mocked (in-memory) adapter

Reply:
The intent was to be able to debug analytics without using the mixpanel adapter and sending events from local, but we can create a new adapter for debug logging if needed.

Removed log from fallback implementation and using log adapter in mixpanel adapter to log errors.